Gems/Multiplayer/Code/Source/ReplicationWindows/NullReplicationWindow.cpp

```diff
@@ -21,7 +21,7 @@ namespace Multiplayer
 
     uint32_t NullReplicationWindow::GetMaxEntityReplicatorSendCount() const
     {
-        return 0;
+        return 128;
```

Comment:
You might move this out to a constant with a nice name and a comment - something like "MaxAutonomousEntitiesToUpdate" since it isn't clear why we would ever updating entities on a NULL window.

Reply:
Alternatively, on Crucible, that number represented the max proxies to send, and autonomous entities did not count against this cap.

Reply:
I like that idea!  Did Crucible allow autonomous entity updates in both directions? (allowing auth to always send properties to auton as well as auton always sending properties to auth).  Check out my update :)

Reply:
Crucible didn't use the Autonomous->Authority updates, and those were turned off at some point.  On the server, when replicating proxies, the Authority->Autonomous updates also did not count against the the cap of entity updates for a send (since those are always critical).  So yes, updating that name to be GetMaxProxyEntityReplicatorSendCount() is a good idea.

I should also mention that this version of the code in O3DE has a few latent issues remaining that were fixed for Crucible.  One major one is that if we have a large number of entities to update, that includes entity creations, those could be starved due to other entities trying to update their properties.  For example, if there was a herd of creatures constantly moving, those would always want to send data, and those updates could prevent the server from sending entity creation updates (since there's no differentiation in priority between those two cases).  I would recommend grabbing some of those fixes.

Reply:
fixed!